flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogTest.java

```diff
@@ -154,4 +161,13 @@ protected void checkEquals(CatalogView v1, CatalogView v2) {
 		// thus properties of Hive view is a super set of those in its corresponding Flink view
 		assertTrue(v2.getProperties().entrySet().containsAll(v1.getProperties().entrySet()));
 	}
+
+	@Override
+	protected void checkEquals(CatalogPartition expected, CatalogPartition actual) {
+		HiveCatalogPartition hivePartition1 = (HiveCatalogPartition) expected;
```

Comment:
We probably want to make sure the casting is valid before we do so. Better yet, we need to make sure the classes of the inputs are also equal.

Reply:
OK